packages/create-preact-app/lib/symbols.js

```diff
@@ -0,0 +1,15 @@
+const { blue, yellow, red } = require('kleur');
+
+const main = {
+	info: blue('ℹ'),
+	warning: yellow('⚠'),
+	error: red('✖'),
+};
+
+const win = {
+	info: blue('i'),
+	warning: yellow('‼'),
+	error: red('×'),
+};
+
+module.exports = process.platform === 'win32' ? win : main;
```

Comment:
Items like this (that are duplicated) might need suggestions from someone who's more experienced with mono-repos. Not sure if there's a better solution without introducing another package that holds some of this common stuff?